### PR TITLE
Feature hello

### PR DESCRIPTION
**Pull request by Obote**
### What does this PR do?

- Use Figma design to create a UI.
- Use Flexbox to place elements on the page.
- Use CSS Grid to place elements on the page.
- Build a personal portfolio site.
- Configure Linters
- Update README file

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
# Hi @Obote😃, 

As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.

## **Invalid Submission**

- Please make sure that your PR's linter's checks are all passing in order to get it reviewed.
_Kindly notice that it's nearly impossible to review un-linter-checked PR._

_Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

<hr>

_Invalid Code Review Request does not count into the code reviews limit._

**Comment by MahmoudBakr23** [comment]
> # Hi @Obote😃,
> As much as I appreciate your hard work on this project, I'm afraid that I can't perform a review on this pull request.
> 
> ## **Invalid Submission**
> * Please make sure that your PR's linter's checks are all passing in order to get it reviewed.
>   _Kindly notice that it's nearly impossible to review un-linter-checked PR._
> 
> _Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready._ 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
> 
> _Invalid Code Review Request does not count into the code reviews limit._

Please fix the linter errors to get your PR reviewed

File: README.md [hunk 1]
```diff
@@ -1,59 +1,37 @@
...
-- Major languages
-- Frameworks
-- Technologies used
-
+- node
+- hmtl
+- css
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
```
Comment (MahmoudBakr23) [inline README.md:19]: **Highlight**
Extra credit for the live demo link 💯 keep rocketing 🚀

File: README.md [hunk 2]
```diff
@@ -1,59 +1,37 @@
...
-# Project Name
+## My portfolio mobile version
+
+This project is a mobile version of my portfolio. It will be continous project that will show what I am capable of doing a developer.
+
+Project day 3 of Microserve bootcamp, module 1
 
-> One paragraph statement about the project.
 
 ![screenshot](./app_screenshot.png)
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Please fix the screenshot of the project in order to keep your repo professional.
_Please keep in mind that such a small mistake might leave a bad impression on future viewers._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                <li><a href="#"><img src="./assets/icons/medium.png" alt="Medium"></a></li>
+            </ul>
+                <a class="arrow" href="#"><img src="./assets/icons/element-right.png" alt="arrow"></a>   
+            </div>   
+            <div class="experience-title">
+                <h2>MY RECENT <br> WORK</h2>
+            </div> 
+             
+        </section>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Kindly remove the extra blank lines and indentations for a cleaner code file.
_Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart._
> You can read about this note and other useful ones [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>    
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>  
+                <li>Ruby</li>
+            </ul>
+
+            <button type="button" class="project-button">See Project</button>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Kindly wrap all the projects inside a `ul` tag and `li` tag per project.
_Following the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) is very important for the website's reading and optimization._
**Hint**
```
<section>
  <ul>
    <li>
      // single project's elements here
    </li>
    <li>
      // single project's elements here
    </li>
    <li>
      // single project's elements here
    </li>
  </ul>
</section> // end of the project's section container
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            <img src="./assets/images/bg.jpg" alt="project image">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>    
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>  
+                <li>Ruby</li>
+            </ul>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._

File: index.html (hunk 4 above)
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css <img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>          
+                <li>bootstrap <img src="./assets/images/vert-line.png" alt="vertical line"></li>         
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>html<img src="./assets/images/vert-line.png" alt="vertical line"></li>              
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="stack-container">
+                <li>css<img src="./assets/images/vert-line.png" alt="vertical line"></li>             
+                <li>htm<img src="./assets/images/vert-line.png" alt="vertical line">l</li>
+                <li>bootstrap<img src="./assets/images/vert-line.png" alt="vertical line"></li>              
+                <li>Ruby</li>
+            </ul>
+            <button type="button" class="project-button">See Project</button>
+
```
Comment (MahmoudBakr23) [inline index.html]: - [ ] Please delete the extra blank line here
_For more details about this requirement, please see the comment above._

File: style.css [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+  justify-content: space-between;
+  color: #fff;
+  padding-top: 48px;
+}
+
+.experience-title {
+  padding-top: 83.85px;
+}
+
+.experience-title h2 {
```
Comment (MahmoudBakr23) [inline style.css:172]: - [ ] Kindly follow the given design template and fix this title's spacings (margin and padding)
_Please keep in mind that you should follow the original design's typography, font size, font weight, colors, images, and spacings in order to provide a professional, consistent, and good-looking portfolio.
### **Your design**
Screenshot
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)**
Screenshot1

File: style.css [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #9490ff;
+}
+
+.experience-container > img {
```
Comment (MahmoudBakr23) [inline style.css:183]: - [ ] Please make sure that there is one `<section>` tag is wrapping the whole work section. _Kindly see the hint I left in the above comment._

- [ ] However, make sure that you're following the given design in all aspects (_See above comment for more details._) You should make the background image of the work section as the given design.
Please refer to the design template, then make the corresponding changes.

- [ ] Following the two previous required changes, please keep in mind that it's mandatory in the [project's requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to use both CSS grid and flexbox to align the work's section.
**Hint**
```
.work-section { // the whole container
  display: grid;
}

.project { // single project's container
  display: flex;
}
```

File: style.css [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+  height: 40px;
+  width: 312px;
+  left: 0;
+  top: 357px;
+  border-radius: 0;
+  padding-top: 24px;
+  align-items: center;
+}
+
+.stack-container li {
```
Comment (MahmoudBakr23) [inline style.css:229]: - [ ] Kindly fix the styles of these list items to match the given design.

### **Your design**
Screenshot2
### **Original design**
Screenshot3

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Obote 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 💯 
- Descriptive PR headline, summary, and commit messages 💯 
- Good use of Semantic HTML concepts 💯 
- Live Demo Link 💯 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄


 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
Hello @Obote,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**